### PR TITLE
NEWS section for 3.0.0-rc1

### PR DESCRIPTION
**Pull request by rbott**
This PR adds a new section to thew NEWS file to prepare the 3.0.0-rc1 release.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -2,6 +2,56 @@ News
...
+
+Other notable changes
+~~~~~~~~~~~~~~~~~~~~~
+
+Bugfixes:
+ - Fix distribution of hmac.key to new nodes - this has been pulled from the 2.17 tree #(1494)
+
+Compatibility Improvements:
+ - Open vSwitch: Do not fail to add node when the ovs_name or ovs_link already exists (#1495)
+ - Improved support for DRBD => 8.4 (#1496)
```
Comment (apoikos) [inline NEWS:39]: `=>` → `>=` :-)

**Review by apoikos — APPROVED** [review]
Other than the small typo, LGTM, thanks!

**Review by iustin — APPROVED** [review]
Looks good, thanks!

File: NEWS (hunk 1 above)
Comment (iustin) [reply NEWS:39]: Or ≥ ? (somewhat joking, just that I'm an Unicode fan)

**Comment by iustin** [comment]
By the way, I've triggered a rebuild. One the haskell tests failed due to known problems with quickcheck tests…

**Comment by mbakke** [comment]
Perhaps also mention #1503? Though I suppose it's not terribly interesting for the average user.

**Comment by saschalucas** [comment]
friendly ping @rbott: could you please update and merge your NEWS PR (see @apoikos comment and maybe mention #1503 from @mbakke). Without NEWS a release can not be done.

If someone else should do it, please send a note here.

**Comment by rbott** [comment]
Oh I am sorry, I entirely forgot to merge this. I will not be near a computer for the next six days. Please merge it for me Apollon or Sascha!


Am 17. September 2020 18:20:05 MESZ schrieb saschalucas <notifications@github.com>:
>friendly ping @rbott: could you please update and merge your NEWS PR
>(see @apoikos comment and maybe mention #1503 from @mbakke). Without
>NEWS a release can not be done.
>
>If someone else should do it, please send a note here.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/ganeti/ganeti/pull/1519#issuecomment-694342782

--

Mit freundlichen Grüßen / With Kind Regards
  Rudolph Bott

**Comment by saschalucas** [comment]
> ... Please merge it for me Apollon or Sascha!

Done. Pimped NEWS making `check-news` happy, fixed DRBD version relation symbol, mention #1503.

**Pull request merged**